wp-seo-main.php

```diff
@@ -260,9 +260,10 @@ function wpseo_init() {
  * Used to load the required files on the plugins_loaded hook, instead of immediately.
  */
 function wpseo_frontend_init() {
-	add_action( 'init', 'initialize_wpseo_front' );
+	WPSEO_Frontend::get_instance();
```

Comment:
Fair point well made! But that renders [`initialize_wpseo_front`](https://github.com/Yoast/wordpress-seo/blob/trunk/inc/wpseo-functions.php#L12-L19) obsolete. Please deprecate it and move it [down](https://github.com/Yoast/wordpress-seo/blob/trunk/inc/wpseo-functions.php#L401).